src/lib.rs

```diff
@@ -3,10 +3,12 @@
 
 //! Render a Lottie animation to a Vello [`Scene`](crate::vello::Scene).
 //!
-//! However, this is also intended to be the preferred integration between Vello and [Lottie](https://lottie.github.io/lottie-spec/), so [consider
-//! contributing](https://github.com/linebender/velato) if you need a feature which is missing.
+//! This currently lacks support for a [number of important](crate#unsupported-features) SVG features.
```

Comment:
How are Velato/Lottie and SVG related?

Reply:
This is a typo, probably since we copied this part from vello_svg.